libbuteosyncfw/common/Logger.cpp

```diff
@@ -206,7 +206,7 @@ void Logger::write(int aLevel, const char *aMsg)
     {
         if(aLevel >= QtCriticalMsg)
         {
-            syslog(LOG_CRIT, sysLogMsg.toLocal8Bit().data());
+            syslog(LOG_CRIT, sysLogMsg.toLocal8Bit().data(), "");
```

Comment:
This appears kind of bypassing the security warning. Maybe rather syslog(LOG_CRIT, "%s", syslogMsg...)?